pkg/aws/awscontrolplane/mutate_awscontrolplane.go

```diff
@@ -78,10 +78,16 @@ func (m *Mutator) MutateCreate(request *admissionv1.AdmissionRequest) ([]mutator
 	if _, _, err := mutator.Deserializer.Decode(request.Object.Raw, nil, awsControlPlaneCR); err != nil {
 		return nil, microerror.Maskf(parsingFailedError, "unable to parse awscontrol plane: %v", err)
 	}
-	releaseVersion, err := releaseVersion(awsControlPlaneCR)
+
+	patch, err = m.MutateReleaseVersion(*awsControlPlaneCR)
 	if err != nil {
-		return nil, microerror.Maskf(parsingFailedError, "unable to parse release version from AWSControlPlane")
+		return nil, microerror.Mask(err)
 	}
+	releaseVersion, err := releaseVersion(awsControlPlaneCR, patch)
```

Comment:
The patch is included here so that we get the release version of the cluster even if it's not set on the CR yet.